WcaOnRails/db/structure.sql

```diff
@@ -1053,8 +1053,9 @@ INSERT INTO schema_migrations (version) VALUES ('20160727000015');
 
 INSERT INTO schema_migrations (version) VALUES ('20160731181145');
 
-INSERT INTO schema_migrations (version) VALUES ('20160831212003');
-
 INSERT INTO schema_migrations (version) VALUES ('20160811013347');
 
+INSERT INTO schema_migrations (version) VALUES ('20160831212003');
```

Comment:
Migrations were out of order, I think. My migration was from the 11th, and you made a new one on the 31st. Probably when rebasing they got swapped.